.ci/validation.cmd

```diff
@@ -32,6 +32,17 @@ if "%OPTION%" ==  "site_without_verify" (
   goto :END_CASE
 )
 
+if "%OPTION%" == "git_diff" (
+  for /f "delims=" %%a in ('git status ^| findstr /c:"Changes not staged"') do set output=%%a
+    if defined output (
+      echo Please clean up or update .gitattributes file.
```

Comment:
IF block is not part of FOR, please put them on same vertical indentation.

Reply:
done